### PR TITLE
 split sq lite db by username

### PR DESCRIPTION
**Pull request by CharlesCCC**
### Split SQLite by Username 
If you experience issue with multiple accounts Instapy.db lockup. You can add the following flag

`-sdb` when running in Command line

or 

To do this simply pass the `split_db=True` parameter in the InstaPy constructor like so:

```python
session = InstaPy(username=insta_username,
                  password=insta_password,
                  headless_browser=False,
		  split_db=True,
                  multi_logs=True)
```

### CODE REVIEW

File: DOCUMENTATION.md [hunk 1]
```diff
@@ -2674,6 +2675,24 @@ Settings.database_location = '/path/to/instapy.db'
...
+
+or 
+
+To do this simply pass the `split_db=True` parameter in the InstaPy constructor like so:
+
+```python
+session = InstaPy(username=insta_username,
+                  password=insta_password,
+                  headless_browser=False,
+		              split_db=True,
```
Comment (ishandutta2007) [inline DOCUMENTATION.md]: indentation mismatch

**Review by ishandutta2007 — COMMENTED** [review]
the changes look good to me. but since you have set split_db to True as default(which I believe is right value for long run) but for now we should send out a note to existing users that it would flush their history and create new db once they update.

File: DOCUMENTATION.md (hunk 1 above)
Comment (CharlesCCC) [reply DOCUMENTATION.md]: actually, the default value is False, this is the documentation to show people how to use it.

Comment (ishandutta2007) [reply DOCUMENTATION.md]: oh mad bad

**Review by ishandutta2007 — APPROVED** [review]
if False it can be merged immediately

**Review by ishandutta2007 — APPROVED** [review]

**Review by timgrossmann — APPROVED** [review]

**Comment by timgrossmann** [comment]
@CharlesCCC @ishandutta2007 Thank you very much guys for making this happen 👍!

**Comment by GaWr26** [comment]
Thanks for this guys but I seem to be missing something. I'm on 0.4.1 and if I add 

`split_db=True` to the constructor I get the following error:

`session = InstaPy(username=insta_username, password=insta_password, multi_logs=True, disable_image_load=True, headless_browser=True, split_db=True)
TypeError: __init__() got an unexpected keyword argument 'split_db'`

**Comment by CharlesCCC** [comment]
@flashguy22 I'm not able to reproduce:

Here is my quickstart.py file

```python 
# set workspace folder at desired location (default is at your home folder)
set_workspace(path=None)

# get an InstaPy session!
session = InstaPy(username='username', password='123', multi_logs=True, disable_image_load=True, headless_browser=True, split_db=True)

with smart_run(session):
    """ Activity flow """
    # general settings
    session.set_dont_include(["friend1", "friend2", "friend3"])

    # activity
    session.like_by_tags(["natgeo"], amount=1)
```

**Comment by GaWr26** [comment]
My bad, I had a version installed with pip and one with pip3.
Can't wait for this to be cleaned up later this year...

Thanks @CharlesCCC for the quick reply!

**Pull request merged**